asmcli/asmcli.sh

```diff
@@ -1971,16 +1987,16 @@ is_membership_crd_installed() {
 }
 
 populate_environ_info() {
```

Comment:
Since you were super detailed and renamed all of the other occurrences, can we rename this function as well?